app/routes/authentication.py

```diff
@@ -84,15 +84,22 @@ def register_user():
     if not email and password:
         return json_api(BadRequest, ErrorSchema), 400
 
-    newuser = User(email=email, confirmed=False, tier="paid")
+    reg_user_count = User.query.filter_by(confirmed=True).count()
+    if reg_user_count > 1000:
+        new_user = User(email=email, confirmed=False, tier="waiting")
+    else:
+        new_user = User(email=email, confirmed=False, tier="paid")
 
     try:
-        newuser.set_password(password)
-        db.session.add(newuser)
+        new_user.set_password(password)
+        db.session.add(new_user)
         db.session.flush()
-        authentication.send_registration_email(email)
     except IntegrityError:
         return json_api(UnprocessableEntity, ErrorSchema), 422
+    if reg_user_count > 1000:
```

Comment:
This feels like the wrong place for this logic.

I think we should consider something like

`authentication.send_registration_email(user.id)`

And then have _that_ method determine what email to send.